website/src/components/subscriptions/notificationChannels/NotificationChannelDisplay.tsx

```diff
@@ -0,0 +1,263 @@
+                <div className='modal-box p-0'>
+                    <div className='flex flex-col items-center gap-2 bg-gray-200 p-6'>
+                        <div className='iconify size-12 mdi--slack'></div>
+                        <h2 className='text-lg font-bold'>Add Slack channel</h2>
+                    </div>
+
+                    <div className='p-6'>
+                        <div>
+                            <h3 className='mb-2'>Slack webhook URL</h3>
+                            <p className='mb-2 text-sm text-gray-400'>
+                                You can get the Slack Incoming Webhook URL in the Slack settings{' '}
+                                <span>
+                                    {' '}
+                                    Channel Settings{'>'}Add an App{'>'}Incoming WebHooks.
+                                </span>
+                            </p>
+                            <label className='w-full'>
+                                <input
+                                    className='input input-sm input-bordered w-full'
+                                    placeholder='https://hooks.slack.com/services/...'
+                                />
+                            </label>
+                        </div>
+
+                        <div className='divider'></div>
+
+                        <div>
+                            <h3 className='mb-2'>Name of channel</h3>
+                            <p className='mb-2 text-sm text-gray-400'>
+                                You can provide a custom name for you channel to find it later on.
+                            </p>
+                            <label className='w-full'>
+                                <input className='input input-sm input-bordered w-full' placeholder='Name' />
+                            </label>
+                        </div>
+
+                        <div className='divider'></div>
+
+                        <div className='modal-action'>
+                            <form method='dialog'>
+                                <button className={'btn btn-outline float-right w-24'}>Discard</button>
+                                {/*TODO: Are you sure you want to discard message.*/}
+                            </form>
+                            <form method='dialog'>
+                                <button className={'btn btn-primary float-right w-24 underline'}>Add</button>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </dialog>
+        </>
+    );
+}
+
+function EditEntry() {
+    return (
+        <button className='btn btn-sm'>
+            <div className='iconify mdi--pencil' />
+        </button>
+    );
+}
+
+function DeleteEntry() {
+    return (
+        <button className='btn btn-ghost btn-sm'>
+            <div className='iconify mdi--trash-can-outline' />
+        </button>
+    );
+}
```

Comment:
A tooltip would be nice (since those buttons don't have a textual description in the button itself)